content/deploy-troubleshooting.md

```diff
@@ -4,36 +4,51 @@ order: 15
 description: Learn how to troubleshoot your deploy and get answers to frequently asked questions
 ---
 
-<h2 id="503-errors">503 errors</h2>
+<h2 id="general-advice">General Advice</h2>
 
-The 503 error ("Service Unavailable: No healthy enpoints to handle the request") indicates that our image builder was not able to successfully create a container, when attempting to deploy. This usually indicates a problem in your code that prevents deployment. You may also see this error temporarily, if trying to resolve a URL in your browser within 1-2 minutes of issuing your deploy command.
+Check these items if you're having trouble with uptime, performance or deployment.
+- Your Meteor version. More current versions may resolve issues found in older Meteor versions. While the reverse is rare and generally shouldn't happen, if you're recently upgraded Meteor versions and start having difficulties, consider reverting to your last Meteor version.
+- Your container's memory and CPU usage. If your app is running out of memory, you may need to either switch to a larger container or refactor your app to use less memory. In the short term, the only guaranteed solution is to use a container large enough to handle all your app's memory needs.
+- Your app's logs. While 'All' shows all errors, consider breaking it down by tab. If your app is running and struggling before failing, check the 'App' tab. If your app is crashing before deployment and never successfully launching, check the 'Service' tab. Note that both can happen simultaneously: the earlier version of your app may be throwing errors, while the recent version created to fix the problem may have a code error preventing deployment. In that case, 'Service' will be more helpful.
+- Check GitHub to see if your issue is listed with a suggested workaround or solution.
+- Consider running more than 1 container, if uptime for your app is mission-critical and you're currently running a single container.
```

Comment:
or can suggest 3 which is "HA"

could be more explicit: "If you only run one container, that means that the machine your container is running on is a single point of failure; in the case of a hardware problem, your app will be down until Galaxy starts it up again on a new machine."